propose-freeform/propose-vote-enact-freeform.py

```diff
@@ -0,0 +1,279 @@
+    node_url_rest = os.getenv("NODE_URL_REST")
+    if not helpers.check_url(node_url_rest):
+        print("Error: Invalid or missing NODE_URL_REST environment variable.")
+        sys.exit(1)
+
+    wallet_server_url = os.getenv("WALLETSERVER_URL")
+
+    wallet_name = os.getenv("WALLET_NAME")
+    if not helpers.check_var(wallet_name):
+        print("Error: Invalid or missing WALLET_NAME environment variable.")
+        sys.exit(1)
+
+    wallet_passphrase = os.getenv("WALLET_PASSPHRASE")
+    if not helpers.check_var(wallet_passphrase):
+        print("Error: Invalid or missing WALLET_PASSPHRASE environment variable.")
+        sys.exit(1)
+
+    # Help guide users against including api version suffix on url
+    wallet_server_url = helpers.check_wallet_url(wallet_server_url)
+
+    #####################################################################################
+    #                           W A L L E T   S E R V I C E                             #
+    #####################################################################################
+
+    print(f"Logging into wallet: {wallet_name}")
+
+    # __login_wallet:
+    # Log in to an existing wallet
+    req = {"wallet": wallet_name, "passphrase": wallet_passphrase}
+    response = requests.post(f"{wallet_server_url}/api/v1/auth/token", json=req)
+    helpers.check_response(response)
+    token = response.json()["token"]
+    # :login_wallet__
+
+    assert token != ""
+    print("Logged in to wallet successfully")
+
+    # __get_pubkey:
+    # List key pairs and select public key to use
+    headers = {"Authorization": f"Bearer {token}"}
+    response = requests.get(f"{wallet_server_url}/api/v1/keys", headers=headers)
+    helpers.check_response(response)
+    keys = response.json()["keys"]
+    pubkey = keys[0]["pub"]
+    # :get_pubkey__
+
+    assert pubkey
+    print("Selected pubkey for signing")
+
+    #####################################################################################
+    #                              F I N D   A S S E T S                                #
+    #####################################################################################
+
+    # __get_assets:
+    # Request a list of assets available on a Vega network
+    url = f"{node_url_rest}/assets"
+    response = requests.get(url)
+    helpers.check_response(response)
+    # :get_assets__
+
+    # Debugging
+    # print("Assets:\n{}".format(
+    #    json.dumps(response.json(), indent=2, sort_keys=True)))
+
+    #####################################################################################
+    #                   G O V E R N A N C E   T O K E N   C H E C K                     #
+    #####################################################################################
+
+    # Get the identifier of the governance asset on the Vega network
+    assets = response.json()["assets"]
+    vote_asset_id = next((x["id"] for x in assets if x["details"]["symbol"] == "tVOTE"), None)
```

Comment:
tVOTE is gone now, it's VEGA (testnet) - but maybe it should just be 'VEGA' so it's applicable to non-testnet too